app/components/MazemapEmbed/index.js

```diff
@@ -90,7 +153,35 @@ export const MazemapEmbed = ({ mazemapPoi, ...props }: Props) => {
   }
   return (
     <>
-      <div style={{ height: props.height || 400 }} id="mazemap-embed" />
+      <div
+        style={{
+          height: props.height || 400,
+          opacity: blockScrollZoom || blockTouchMovement ? 0.5 : 1,
+          touchAction: 'pan-x pan-y',
+        }}
+        id="mazemap-embed"
+      >
+        {(blockScrollZoom || blockTouchMovement) && (
+          <span
+            style={{
+              fontSize: '1.5rem',
+              textAlign: 'center',
+              color: '#111',
+              position: 'absolute',
+              top: '50%',
+              left: '50%',
+              width: '100%',
+              transform: 'translate(-50%,-50%)',
+              zIndex: 5,
+              pointerEvents: 'none',
+            }}
+          >
+            {blockScrollZoom
+              ? 'Hold ctrl for å zoome'
```

Comment:
Is it ctrl for mac as well?  :O

Reply:
Why must you people always be like this

Reply:
> Is it ctrl for mac as well? :O

Yes, or you could have needed a special case for macs.

Reply:
command definitely feels more natural than ctrl on mac. I think it's worth adding a special case.